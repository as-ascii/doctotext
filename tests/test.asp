<!doctype html>
<html>
  <head>
    <title>Example Page</title>
  </head>
  <body>
<%
  'This is where the ASP code begins
  'ASP will generate the HTML that is passed to the browser
  'A single quote denotes a comment, so these lines are not executed
  'Since this will be HTML, we included the html and body tags
  'for Classic ASP we use Response.Write() to output our text
  'like this
  
  Response.Write ("Hello world")
  
  'Now we will end the ASP block and close our body and html tags
%>
  </body>
</html>
