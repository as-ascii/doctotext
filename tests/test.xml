<?xml version = "1.0">
<test>
	<data>Hello World!</data>
</test>
