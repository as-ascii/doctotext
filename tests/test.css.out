body:before {
	content: "Hello Word!";
	color: green;
}

